core/controllers/acl_decorators.py

```diff
@@ -1739,6 +1741,13 @@ def test_can_edit_story(self, topic_id, **kwargs):
                 credentials to edit a story belonging to a
                 given topic.
         """
+        story_domain.Story.require_valid_story_id(story_id)
```

Comment:
Tests? (Add to acl_decorators_test.py.)

Reply:
Oh, I added the test to story_editor.py that checks these, will add here as well.

Reply:
Done